CgfConverter/CgfConverter.csproj

```diff
@@ -11,7 +11,7 @@
   <ItemGroup>
     <PackageReference Include="BCnEncoder.Net" Version="2.1.0" />
     <PackageReference Include="BCnEncoder.Net.ImageSharp" Version="1.1.1" />
-    <PackageReference Include="SixLabors.ImageSharp" Version="2.1.7" />
+    <PackageReference Include="SixLabors.ImageSharp" Version="2.1.8" />
```

Comment:
I feel like there was a bug when I tried to upgrade these in the past.  It's just a patch version so it shouldn't cause any issues, but... let me double check.

Reply:
Let me know and I can always revert it.

Reply:
Everything seemed to work fine.  Not sure what the issue was before but it could be me getting confused.  That happens a lot. :)